### PR TITLE
Small fix

### PR DESCRIPTION
**Pull request by ValentinaHotsa**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -36,6 +36,7 @@
     "redux-persist": "^6.0.0",
     "styled-components": "^6.1.0",
     "sweetalert": "^2.1.2",
```
Comment (ReginaVarybrus) [inline package.json]: Do we need This one if we added `sweetalert2`?

File: src/components/Buttons/FaceBook/FaceBookButton.styled.js [hunk 2]
```diff
@@ -13,31 +12,29 @@ export const FacebookButton = styled.button`
...
     letter-spacing: 0em;
     text-align: center;
   }
-  
+
   &:hover {
     background-color: var(--color-blue-1);
   }
 
   @media (min-width: 768px) {
```
Comment (ReginaVarybrus) [inline src/components/Buttons/FaceBook/FaceBookButton.styled.js]: Let's use `device` constants.

File: src/components/Buttons/GoogleButton/GoogleButton.styled.js [hunk 3]
```diff
@@ -12,31 +12,28 @@ export const GoogleButton = styled.button`
...
     letter-spacing: 0em;
     text-align: center;
   }
-  
+
   &:hover {
     background-color: var(--color-blue-1);
   }
 
   @media (min-width: 768px) {
```
Comment (ReginaVarybrus) [inline src/components/Buttons/GoogleButton/GoogleButton.styled.js]: The same

File: src/components/Buttons/LargeFilled/LargeFilledButtonStyled.js [hunk 4]
```diff
@@ -2,27 +2,26 @@ import { device } from '@/constants/mediaQueries';
...
-text-align: center;
-font-size: 16px;
-background-color: var(--color-brand-blue);
-cursor: pointer;
-white-space: nowrap;
+  padding: 12px 24px;
+  border-radius: 8px;
+  border: 1px solid var(--color-brand-blue);
+  color: var(--white-color);
+  font-family: 'Roboto', sans-serif;
```
Comment (ReginaVarybrus) [inline src/components/Buttons/LargeFilled/LargeFilledButtonStyled.js]: Do we need here font-family if we have it in the GlobalStyles?

File: src/components/Buttons/LargeOutlinedWIcon/LargeOutlinedWIconButtonStyled.js [hunk 5]
```diff
@@ -2,29 +2,27 @@ import { device } from '@/constants/mediaQueries';
...
-cursor: pointer;
-white-space: nowrap;
-    ${({ $mobilestyles }) => $mobilestyles};
+  width: 148px;
+  height: 44px;
+  padding: 12px 24px;
+  border-radius: 8px;
+  border: 1px solid var(--color-brand-blue);
+  color: var(--color-grey-8);
+  font-family: 'Roboto', sans-serif;
```
Comment (ReginaVarybrus) [inline src/components/Buttons/LargeOutlinedWIcon/LargeOutlinedWIconButtonStyled.js]: The same

File: src/components/Buttons/LargeOutlinedWIcon/LargeOutlinedWIconButtonStyled.js [hunk 6]
```diff
@@ -2,29 +2,27 @@ import { device } from '@/constants/mediaQueries';
...
+  font-weight: 700;
+  line-height: 19.2px;
+  text-align: center;
+  font-size: 14px;
```
Comment (ReginaVarybrus) [inline src/components/Buttons/LargeOutlinedWIcon/LargeOutlinedWIconButtonStyled.js:11]: Maybe we have to move all fonts properties fo the buttons to the Global styles?

**Review by ReginaVarybrus — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (ValentinaHotsa) [reply package.json]: done

File: src/components/Buttons/FaceBook/FaceBookButton.styled.js (hunk 2 above)
Comment (ValentinaHotsa) [reply src/components/Buttons/FaceBook/FaceBookButton.styled.js]: done

File: src/components/Buttons/GoogleButton/GoogleButton.styled.js (hunk 3 above)
Comment (ValentinaHotsa) [reply src/components/Buttons/GoogleButton/GoogleButton.styled.js]: done

File: src/components/Buttons/LargeFilled/LargeFilledButtonStyled.js (hunk 4 above)
Comment (ValentinaHotsa) [reply src/components/Buttons/LargeFilled/LargeFilledButtonStyled.js]: done

File: src/components/Buttons/LargeOutlinedWIcon/LargeOutlinedWIconButtonStyled.js (hunk 5 above)
Comment (ValentinaHotsa) [reply src/components/Buttons/LargeOutlinedWIcon/LargeOutlinedWIconButtonStyled.js]: done

File: src/components/Buttons/LargeOutlinedWIcon/LargeOutlinedWIconButtonStyled.js (hunk 6 above)
Comment (ValentinaHotsa) [reply src/components/Buttons/LargeOutlinedWIcon/LargeOutlinedWIconButtonStyled.js:11]: done

**Review by ReginaVarybrus — APPROVED** [review]

**Pull request merged**